Source/App/EbAppMain.c

```diff
@@ -323,7 +305,7 @@ int main(int argc, char* argv[])
         }
         else {
             printf("Error in configuration, could not begin encoding! ... \n");
-            printf("Run %s -help for a list of options\n", argv[0]);
```

Comment:
The old one is better to avoid the hardcoded name.

Reply:
Addressed in last commit